server/src/test/java/org/opensearch/search/aggregations/AggregationProcessorTests.java

```diff
@@ -157,6 +174,9 @@ private void testPostProcessCommon(String aggs, int numSlices, int expectedGloba
                 .thenReturn(result);
         }
         assertTrue(context.queryResult().hasAggs());
+        if (withProfilers) {
```

Comment:
I think it would be cleaner to redesign this test case as parameterized (w/o profiling) instead of copying test cases

Reply:
Hi @reta, appreciate your review. After further investigation, it has been determined that we only perform global aggregation in the post process here. As a result, I've revised the tests, and there is no longer a need for parameterization.